src/pages/RepoPage/FailedTestsTab/FailedTestsPage/hooks/useTestResultsAggregates/useTestResultsAggregates.tsx

```diff
@@ -12,15 +12,17 @@ const TestResultsAggregatesSchema = z.object({
       repository: z.discriminatedUnion('__typename', [
         z.object({
           __typename: z.literal('Repository'),
-          testResultsAggregates: z.object({
-            totalDuration: z.number(),
-            totalDurationPercentChange: z.number().nullable(),
-            slowestTestsDuration: z.number(),
-            slowestTestsDurationPercentChange: z.number().nullable(),
-            totalFails: z.number(),
-            totalFailsPercentChange: z.number().nullable(),
-            totalSkips: z.number(),
-            totalSkipsPercentChange: z.number().nullable(),
+          testAnalytics: z.object({
+            testResultsAggregates: z.object({
+              totalDuration: z.number(),
+              totalDurationPercentChange: z.number().nullable(),
+              slowestTestsDuration: z.number(),
+              slowestTestsDurationPercentChange: z.number().nullable(),
+              totalFails: z.number(),
+              totalFailsPercentChange: z.number().nullable(),
+              totalSkips: z.number(),
+              totalSkipsPercentChange: z.number().nullable(),
+            }),
```

Comment:
i believe testAnalytics can be `.nullable()` here

Reply:
Great point, updated in [1ecbecd](https://github.com/codecov/gazebo/pull/3379/commits/1ecbecdea8ae8f4c0e0ccf2f0b75e6b55aa927cf)